initialize.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/no-unresolved
```

Comment:
this override doesn't need to be here if the build process has ran, does it?

Reply:
it's true, but... it hasn't at the time of the lint call.